tools/clang/test/CodeGenSPIRV/texture.sample-invalid-implicit-lod.hlsl

```diff
@@ -1,30 +0,0 @@
-// Run: %dxc -T vs_6_0 -E main
-
-SamplerState gSampler : register(s5);
-
-// Note: The front end forbids sampling from non-floating-point texture formats.
-
-Texture1D   <float4> t1 : register(t1);
-Texture2D   <float4> t2 : register(t2);
-Texture3D   <float4> t3 : register(t3);
-TextureCube <float4> t4 : register(t4);
-
-float4 main(int2 offset: A) : SV_Position {
-// CHECK: sampling with implicit lod is only allowed in fragment shaders
```

Comment:
why is this test removed? I don't see the corresponding `emitError` code being removed

Reply:
I think I removed them: https://github.com/microsoft/DirectXShaderCompiler/pull/3656/files#diff-6b86e3fcb3f49e0a10dd39cc10c7ec49d6c00bf11422c86743ef63a34eac6303
No?

I want to keep the test to check the spirv-val error, but with the current way of running unit test, we cannot check the spirv-val error.
In the pseudo code, the unit test checks
```
if (error expected): run it with -fcgl -Vd
else: run it -fcgl

check 'CHECK:' lines with the output

if (do validation): run spirv-val // <-- NOTE: no way to check spirv-val error
```

Let me open a bug to support the spirv-val error check in the unit test.

Reply:
Sorry. I correctly update the unit test.